spock-specs/src/test/groovy/org/spockframework/smoke/mock/MockingAndBridgeMethods.groovy

```diff
@@ -36,6 +36,113 @@ class MockingAndBridgeMethods extends Specification {
       0
     }
   }
-}
 
+  @Issue('https://github.com/spockframework/spock/issues/122')
+  def "bridge methods that override concrete methods in a supertype are still intercepted - check stubbed values"() {
+    given:
+    def di = Mock(DiscountedItem)
+    def dpi = Mock(DiscountedPublicItem)
+
+    when:
+    di.getId() >> 333   // bridge method
```

Comment:
use block comments instead of normal comments,

```
when: 'mocking bridge method'
di.getId() >> 333

and: 'mocking non-bridge method'
dpi.getId() >> 555